config/ini.go

```diff
@@ -29,15 +29,15 @@ import (
 )
 
 var (
-	defaultSection = "default"   // default section means if some ini items not in a section, make them in default section,
-	bNumComment    = []byte{'#'} // number signal
-	bSemComment    = []byte{';'} // semicolon signal
-	bEmpty         = []byte{}
-	bEqual         = []byte{'='} // equal signal
-	bDQuote        = []byte{'"'} // quote signal
-	sectionStart   = []byte{'['} // section start signal
-	sectionEnd     = []byte{']'} // section end signal
-	lineBreak      = "\n"
+	defaultSection  = "default"   // default section means if some ini items not in a section, make them in default section,
+	bNumComment        = []byte{'#'} // number signal
+	bSemComment        = []byte{';'} // semicolon signal
+	bEmpty         []byte
+	bEqual                  = []byte{'='} // equal signal
+	bDQuote                = []byte{'"'} // quote signal
+	sectionStart      = []byte{'['} // section start signal
+	sectionEnd          = []byte{']'} // section end signal
+	lineBreak            = "\n"
```

Comment:
did you format the code?

Reply:
I thought I did
Will go through it again in few days to see why everything fails.

Reply:
I did, It was not formatted in the first place...

Reply:
but from the diff shows it's not formatted.

Reply:
On it